tox.ini

```diff
@@ -1,7 +1,8 @@
 [tox]
 envlist = py27, py35
+
 [testenv]
-commands = py.test
+commands = py.test src --cov=src --cov-report term-missing
```

Comment:
Thanks for adding this! Great for your testing analysis, and helpful for me when grading.